packages/zsa/src/zod-safe-function.ts

```diff
@@ -551,7 +551,7 @@ export class ZodSafeFunction<
 
     // finally return the error
     return [
-      null,
+      inputRaw,
```

Comment:
I think this breaks `useActionState` (old useFormState). How can I fix it and at the same time cover my case? 


I run `npm run test:playwright` and it doesn't fail becuase is cached somehow